ui/oopsyraidsy/data/05-shb/raid/e8n.js

```diff
@@ -6,9 +6,57 @@
     ko: /^희망의 낙원 에덴: 공명편 \(4\)$/,
   },
   damageWarn: {
+    'Biting Frost': '4DDB', // 270-degree frontal AoE, Shiva
+    'Driving Frost': '4DDC', // Rear cone AoE, Shiva
+    'Frigid Stone': '4E66', // Small spread circles, phase 1
+    'Reflected Axe Kick': '4E00', // Large circle AoE, Frozen Mirror
+    'Reflected Scythe Kick': '4E01', // Donut AoE, Frozen Mirror
+    'Frigid Eruption': '4E09', // Small circle AoE puddles, phase 1
+    'Icicle Impact': '4E0A', // Large circle AoE puddles, phase 1
+    'Axe Kick': '4DE2', // Large circle AoE, Shiva
+    'Scythe Kick': '4DE3', // Donut AoE, Shiva
+    'Reflected Biting Frost': '4DFE', // 270-degree frontal AoE, Frozen Mirror
+    'Reflected Driving Frost': '4DFF', // Cone AoE, Frozen Mirror
   },
   damageFail: {
   },
   triggers: [
+    {
+      id: 'E8N Shining Armor',
+      gainsEffectRegex: gLang.kEffect.Stun,
+      mistake: function(e) {
+        return { type: 'warn', blame: e.targetName, reason: gLang.kEffect.Stun };
+      },
+    },
+    {
+      id: 'E8N Heavenly Strike',
+      damageRegex: '4DD8',
+      deathReason: function(e) {
+        return { type: 'fail', name: e.targetName, reason: 'Pushed off!' };
```

Comment:
Can you make these reasons a `reason: { en: 'Pushed off!' }` style object to make it easier for other people to add translations (and for `find_missing_translations.py` to not miss it)?

Reply:
I'll do that. I'm currently waiting on a queue to test a change I made to the Frost Armor mistake that might cut it down to one trigger. (I forgot that Frost Armor inflicts Thin Ice on all players, so that would be a MUCH cleaner way to handle this.)